app/controllers/orga/seasons_controller.rb

```diff
@@ -45,23 +45,13 @@ def destroy
   end
 
   # # switch_phase: enables developers to easily switch between time dependent settings in views
-  # by opening and closing the corresponding links in the nav bar
+  # by showing the corresponding links in the nav bar
   def switch_phase
     return if Rails.env.production?
-      case params[:option]
-      when 'Proposals'
-        Season::PhaseSwitcher.fake_proposals_phase
-      when 'Application'
-        Season::PhaseSwitcher.fake_application_phase
-      when 'CodingSummer'
-        Season::PhaseSwitcher.fake_coding_phase
-      when 'RealTime'
-        Season::PhaseSwitcher.back_to_reality
-      end
-    redirect_to orga_seasons_path, notice: "We time travelled into the #{params[:option]} phase"
+    Season::PhaseSwitcher.destined(params[:phase])
```

Comment:
I opted for :destined, analogue to Avdi's UserAdder class with :addition method. What I like about :destined/:destination is that is has a 'going somewhere'-connotation. 
Other options: :destination, :chosen, :distribution, :direction 
Now I am totally into Naming Things, working on a blogpost :-)